pkg/standalone/invoke_test.go

```diff
@@ -0,0 +1,108 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package standalone
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInvoke(t *testing.T) {
+	testCases := []struct {
+		name          string
+		errorExpected bool
+		errString     string
+		appID         string
+		method        string
+		lo            ListOutput
+		listErr       error
+		expectedPath  string
+		postResponse  string
+		resp          string
+	}{
+		{
+			name:          "list apps error",
+			errorExpected: true,
+			errString:     assert.AnError.Error(),
+			listErr:       assert.AnError,
+		},
+		{
+			name:          "appID not found",
+			errorExpected: true,
+			appID:         "invalid",
+			errString:     "app ID invalid not found",
+			lo: ListOutput{
+				AppID: "testapp",
+			},
+		},
+		{
+			name:   "appID found successful invoke empty response",
+			appID:  "testapp",
+			method: "test",
+			lo: ListOutput{
+				AppID: "testapp",
+			},
+		},
+		{
+			name:   "appID found successful invoke",
+			appID:  "testapp",
+			method: "test",
+			lo: ListOutput{
+				AppID: "testapp",
+			},
+			expectedPath: "/v1.0/invoke/testapp/method/test",
+			postResponse: "test payload",
+			resp:         "successful invoke",
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
```

Comment:
I don't see `t.Parallel()` or using go-routine here, so and this copy is not necessary for sequential runs (See https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables , and different samples with and without `t.Parallel()` here https://blog.golang.org/subtests )

Reply:
Also didn't see `&tc` in the `t.Run` so I think we are ok without copying.

Reply:
Same here, resolved without objection or change.